pages/Posts.tsx

```diff
@@ -77,19 +102,59 @@ export default function Posts() {
       </div>
     );
   }
+  const handleLikeClick = (post: Post) => {
+    setPosts(
+      posts.map((p) =>
+        p.id === post.id
+          ? { ...p, likes: (p.likes || 1) + 0, liked: !p.liked }
+          : p,
+      ),
+    );
+  };
 
   return (
     <div ref={scrollContainerRef} className="postContainer">
-      <h1>Post Feed</h1>
+      <h2 className="text-[32px]">Post Feed</h2>
       {posts.map((post: Post) => {
+        const user = users.find((user) => user.id === post.id);
+
         return (
           <div key={post.id} className="postWrapper">
+            <div className=" flex justify-between items-center pb-2">
+              <h2 className="font-bold flex flex-col">
+                {user?.name}
+                <span className="text-gray font-thin	"> @{user?.username}</span>
+              </h2>
+              <HiDotsHorizontal />
+            </div>
             <h2 className="postTitle">{post.title}</h2>
```

Comment:
I don't think it's wise to have 2 `h2`'s in the same `div`. Is there a reason you did it?